### DISCUSSION TITLE
Is it possible to keep title bar but remove the rounded corners?

### DISCUSSION THREAD

**Discussion by iwillreku3206** [kovidgoyal/kitty#5879] [Q&A]
Platform: macOS

Hello, everyone!

I just wanted to ask if it is possible to remove the rounded windows corners but keep the title bar on macOS? Already tried `hide_window_decorations yes` but it removed the title bar, which I want to keep.

Thanks!

**Comment by page-down** [comment]
> ... remove the rounded windows corners but keep the title bar ...

That means that kitty needs to draw its own title bar with window decoration  (Or switch to another Cocoa window style, if there is one), which is currently not available.

I remember kitty uses a simple square corner title bar under wayland GNOME. Not sure if you like it.
Again, this is not available under macOS.

**Reply by iwillreku3206** [reply]
I see... thanks a lot!

**Comment by port19x** [comment]
That would need to be configured on a per os basis. 
I have neither a titlebar nor the rounded corners, since I'm on linux with a tiling window manager.
Not much you can do from the kitty side
